pages/automatedtestingpython.md

```diff
@@ -72,10 +72,19 @@ Finally:
 
 ## MacOS/Ubuntu - Installing Testing Development Environment
 ### Installing Nation
-WIP - Needs some love.
+The process is similar to installing Nation for Windows.
 
 ### Installing Python, Pip, Selenium, Geckodriver, and Firefox
-WIP - Needs some love.
+1. Install Python 3 with `brew install python3` or download Python from https://www.python.org/downloads/release/python-352/)
```

Comment:
@duongdo27 

* On Ubuntu 14.04 onwards, Python 3.4 is installed by default. So we can just state the same.  

* Also, maybe we can add instructions about installing Homebrew on Mac if it isn't already installed, which can be done with the following command:

`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`

* (optional/suggestion) Also, one can simply install official python3 [package](https://www.python.org/downloads/) instead of using brew on Mac.

* we can refer cmd window as 'terminal'. for Mac and Ubuntu.